corehq/apps/cloudcare/templates/formplayer/session_list.html

```diff
@@ -15,9 +15,9 @@
       </tbody>
     </table>
   </div>
-  <div class="pull-right">
-      <ul id="sessions-paginator" class="pagination-sm"></ul>
-  </div>
+  <% if (endPage) { %>
```

Comment:
does this need `{% block pagination_templates %} ... {% endblock %}`

Reply:
since this template was splitted into preview and web-apps template when i added `{% block pagination_templates %} ... {% endblock %}` previously it lead to duplicate blocks warning, so i have refactored and combined the template and added this `block` statement. 
here f9b2563fbb3b56c09e977f583e45f31de9d828ab